src/GrafanaDashboardDeploy/tests/DeployToolTests.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using Xunit;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.IO;
+
+namespace DotNet.Grafana.Tests
+{
+    public class DeployToolTests
+    {
+        [Theory]
+        [InlineData("[Vault(secretName)]", "secretName")]
+        [InlineData("[vault(secretName)]", "secretName")]
+        [InlineData("[vault(secret name)]", "secret name")]
+        [InlineData("[vault(secret ☃)]", "secret ☃")]
```

Comment:
I'd escape this using `\u` syntax.  Some tools do weird things with unicode characters. :-)

Reply:
Well that seems like the tool's problem :-) But I recognize that breaking things this way doesn't help us. I've changed it.

Reply:
There are hundreds of tools that touch code, many of which we don't control the source for (or some we lost ages ago), so couldn't update if we wanted to.  It's actually called out in the style guidelines too.

I don't think there are _many_ bad tools these days... but the last thing we'd want is some SDL tool in some release branch to cause a build break because it assumed ASCII.